projenrc/update-schema.ts

```diff
@@ -4,10 +4,8 @@ import * as tjs from 'typescript-json-schema';
 import { SCHEMA_DIR, getGeneratedSchemaPaths, getSchemaDefinition } from './schema-definition';
 import { exec, log } from './util';
 
-export function schemasChanged(): boolean {
```

Comment:
This function runs a `git diff` of the current branch against `main` to detect whether schemas were changed. This works when executed on the PR before merge, but once the PR is merged it will always return no changes. 

Instead, we now compare against the latest tag, which will always differ from `main` during the release workflow.